grafana/grafana
# prometheus: query-field: monaco: implement open-help-by-default

in the prometheus query field, in the monaco-version, when an autocomplete window opens for the list of metrics for example, if you press ctrl+enter there, we open an additional help-window with extra help.
we want this extra-help-window to open by default, when the autocomplete-window opens.

this pull request does that.

there is unfortunately no simple way to do it, the way to do it is described in detail in the source code.

how to test:
- open the prometheus query editor
- start writing a metric-name, so that you get the metric-name-list autocomplete. it should by-defrault open the extra help window too
